adr/0014-home-assistant-supervised.md

```diff
@@ -20,14 +20,25 @@ This method is considered advanced and should only be used if one is an expert i
 
 ### Supported Operating System, System dependencies and versions
 
-Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. We only support FHS 3.0 on the host file system.
+Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. It can be installed in many different ways, one of them is using the script here https://get.docker.com/ another is adding the repo manually like described here https://www.linode.com/docs/guides/how-to-install-docker-ce-on-debian-10/. We only support FHS 3.0 on the host file system.
 
 - Docker CE >= 19.03
 - Systemd >= 239
 - NetworkManager >= 1.14.6
 - AppArmor == 2.13.x (built into the kernel)
 - Debian Linux Debian 10 aka Buster (no derivatives)
 
+### Required System Packages
+The installer script will check for the following pre-required packages:
+
+- systemctl
+- nmcli
+- apparmor_parser
+- docker
+- jq
+- curl
+- dbus-daemon
+
```

Comment:
This is a duplication of what is listed above already?

Reply:
Not really, it's nowehere documented that e.g. `jq` is required, and it's not part of the standard installer of Debian. So if you would follow the req's and install `docker-ce` you will get an error that it's missing. As you say, maybe it should be up to the user to figure out?

Reply:
Yes, if you need this level of guidance, you should not be running this installation method. We strongly recommend using the Home Assistant Operating System or Home Assistant Container installation methods instead.

As said, this is not an installation guide.

Reply:
I totally understand, sorry for wasting your time, it's highly appreciated. The add-on ecosystem is very nice, so for hw that isn't a nuc, odroid or an pi this method is still very valuable. :+1: